BGAnimations/ScreenEvaluation common/Panes/Pane1/Percentage.lua

```diff
@@ -24,8 +27,35 @@ return Def.ActorFrame{
 		Name="Percent",
 		Text=percent,
 		InitCommand=function(self)
+                        if ecfa2021score then self:visible(false) return end
 			self:vertalign(middle):horizalign(right):zoom(0.38)
 			self:x(45)
 		end
-	}
+	},
+	
+	-- ECFA 2021 score
+        LoadFont("Wendy/_wendy white")..{
+                Name="ECFA2021",
+                Text="",
+                InitCommand = function(self)
+                        if not ecfa2021score then return end
+                        self:vertalign(middle):horizalign(right):zoom(0.38):x(40)
+                        self:settext(ECFAPointString(ecfa2021score))
+                end
+        },
+        -- "Insufficient FA" message
```

Comment:
![image](https://user-images.githubusercontent.com/19793145/101170899-5f9e9500-363f-11eb-8b1c-81c816f7efe2.png)

Reply:
This will conflict with longer chart descriptions and personal record text. Please move somewhere else, perhaps into the score box next to the 0, or in the free space above the graph.

Reply:
What if we just hide the 0 in this case and only have the text in the score box? (I'll upload a screenshot asap)

Reply:
![image](https://user-images.githubusercontent.com/19793145/101208634-bd98a000-3672-11eb-8079-38b22ef6fc82.png)

Reply:
looks good to me! 
(discouraging much)